docs/getting-started/publishing-data.md

```diff
@@ -19,19 +19,25 @@ You can read and learn about CSV in [this post](/docs/data-packages/csv).
 
 ## Get some CSV data
 
-You'll need a CSV file to push.
-
 Let's have a simple CSV file. Create a file in your current directory named `mydata.csv` and paste the following into it (you can use your own):
 
+**Note:** *It will validate your data prior publishing, if everything is fine - otherwise you'd get validation errors*
+
 ```
-A,B,C
+number,string,boolean
 1,one,true
 2,two,false
 ```
 
-## Push the data
+## Publish the data
+
+You will need to login (or signup) before publishing:
```

Comment:
Think would be better to just say that they'll need a login and emphasize it's free. instead of giving instructions. Something like
**Note**: *you will need to be logged in to publish data on datahub. It's simple and easy just type `data login` and follow instructions*